lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -54,7 +51,7 @@ class FeatureCard extends React.Component {
   }
 
   getCampaigns(staffPickOnly, paginate, page, callback) {
-    this.requests.push(this.serverRequest = $.get(`${API_URL}campaigns?${staffPickOnly ? 'staff_pick=true&' : ''}${page != -1 ? 'page=' + page : ''}`, result => {
+    this.requests.push(this.serverRequest = $.get(`${this.apiClient.url}campaigns?${staffPickOnly ? 'staff_pick=true&' : ''}${page != -1 ? 'page=' + page : ''}`, result => {
```

Comment:
little confused, wasn't the point of writing the new Api Client to replace these get requests?